docs/operators/operator-node/node-setup/manual-setup.md

```diff
@@ -203,10 +162,14 @@ ssv:
 eth2:
   # HTTP URL of the Beacon node to connect to.
   BeaconNodeAddr: <ETH2_NODE> # e.g. http://example.url:5052
+  # If you want to use multiple endpoints you can divide them with ;
+  # e.g. http://example.url:5052;http://example.url:5053
 
 eth1:
   # WebSocket URL of the Eth1 node to connect to.
   ETH1Addr: <ETH1_WEBSOCKET_ADDRESS> # e.g. ws://example.url:8546/ws
+  # If you want to use multiple endpoints you can divide them with ;
```

Comment:
Please make this more visible with an info box, like you've done on the "main" node setup page. Rest is good